css/amp-story-player-shadow.css

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /** See #23876 and #31363. */
+ :host {
+  all: initial;
+  color: initial;
+}
+
+.story-player-iframe {
```

Comment:
I imagine there might be some side effects of changing these classnames to reflect the new naming, but it would be nice to do if it's simple.

Reply:
Publishers may already be using some of these classnames (ones not prefixed by `i-amphtml-`). 
This is a good callout but I'm not sure we can safely change the calssnames to match the filename.

Reply:
I was afraid of that but wanted to inquire.

Reply:
If it's within a shadow DOM we should be able to safely change it :))